### PR TITLE
Install ENV_BANG, and convert all ENV access to use it

### PR DESCRIPTION
**Pull request by jcamenisch**
Addresses issue #2 
## Tasks
- [x] Handle `default: false` scenario
- [x] Make ENV! read from ENV in realtime, not stored values.
- [x] Create railtie in env_bang gem to automatically require the env.rb file before application.rb loads.

### CODE REVIEW

File: config.ru [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This file is used by Rack-based servers to start the application.
 
 require File.expand_path('config/environment', __dir__)
-use Rack::CanonicalHost, ENV['CANONICAL_HOST'] if ENV['CANONICAL_HOST']
+use Rack::CanonicalHost, ENV!['CANONICAL_HOST'] if ENV!['CANONICAL_HOST']
```
Comment (vlucas) [inline config.ru:4]: Do you need to have the `if` here anymore?

Comment (jcamenisch) [inline config.ru:4]: Yes, because if the variable's not set (or it's set to false/nil), we don't want the middleware at all.

**Comment by jcamenisch** [comment]
OK, this one's actually ready to merge now. It does work from the Rails console.

This approach is still pretty green, so we could consider using this on Kikuchat before merging to the template. Or we could move forward and revise later.

**Comment by alkrauss48** [comment]
I was just about to merge this!

**Comment by jcamenisch** [comment]
Doh! Haha

**Pull request closed without merging**